test/mjsunit/simd/float32x4.js

```diff
@@ -973,3 +972,87 @@ testFloat32ArrayByteOffset();
 testFloat32ArrayByteOffset();
 %OptimizeFunctionOnNextCall(testFloat32ArrayByteOffset);
 testFloat32ArrayByteOffset();
+
+function testFloat32x4fromInt32x4() {
+  var m = SIMD.float32x4(9, 10, 11, 12);
+  var nMask = SIMD.int32x4.fromFloat32x4(m);
+  var n = SIMD.float32x4.fromInt32x4(nMask);
+
+  assertEquals(9.0, n.x);
+  assertEquals(10.0, n.y);
+  assertEquals(11.0, n.z);
+  assertEquals(12.0, n.w);
+};
+
+testFloat32x4fromInt32x4();
+testFloat32x4fromInt32x4();
+%OptimizeFunctionOnNextCall(testFloat32x4fromInt32x4);
```

Comment:
As there is no CrankShaft implementation, leave the OptimizeFunctionOnNextCall in your next patch.